### ISSUE TITLE
[Bug] Column.data_type does not take into account the data_length

### ISSUE THREAD

**Issue by sandhein** [oracle/dbt-oracle#29]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When using column.data_type for columns of type varchar or numeric, the adapters just gives back the column_type, without the size / data_lengths. You can see this behaviour for example when using dbt_utils.union:

![grafik](https://user-images.githubusercontent.com/26199754/177748463-38d409d6-d16b-4219-b163-6235e60c2a6d.png)

Thsi results in ORA-00906: missing left paranthesis

### Expected Behavior

Return the data_lengths/size in parathesis:
![grafik](https://user-images.githubusercontent.com/26199754/177748657-176e5ac6-338e-43f9-a755-4f498840e1a5.png)

This can be done by overiting the following methods:
`
@dataclass
class OracleColumn(Column):

    def is_string(self):
        # in Oracle, 'varchar2' is also a string:
        if self.dtype.lower() in ['varchar2', 'char', 'varchar']:
            return True
        return super().is_string()

    @classmethod
    def string_type(cls, size: int) -> str:
        return "varchar2({})".format(size)

    def is_numeric(self) -> bool:
        # in Oracle, 'number' is also a numeric:
        if self.dtype.lower() in ['number', 'float']:
            return True
        return super().is_numeric()
`

### Environment

```markdown
- OS: Windows
- Python: Python 3.8.10
- dbt: 1.1.1
```

### What Oracle database version are you using dbt with?

19c

**Comment by aosingh** [comment]
Hi @sandhein 

This is fixed now and we plan to release the fix in the upcoming dbt-oracle version 1.1.1. Meanwhile, you can test the fix using the release candidate. It can be installed using pip

```
pip install dbt-oracle==1.1.1rc1
```

Let me know if you have any questions

**Comment by aosingh** [comment]
This is now fixed in `dbt-oracle==1.1.1` Please test it and let me know if you face this issue.
